.github/workflows/testing.yml

```
@@ -25,24 +29,36 @@ jobs:
+            pwd
 
         - name: Create Coverage Report
           run: |
             coverage report -m
             coverage xml
+            ls
 
         - name: DeepSource Test Coverage
           uses: deepsourcelabs/test-coverage-action@master
           with: 
             key: python
```

yamllint (1.35.1)

[warning] wrong indentation: expected 14 but found 12

(indentation)